CocoaPods.md

```diff
@@ -0,0 +1,83 @@
+# Using CocoaPods
+
+When you develop for iOS, you can quickly add third party libraries to your NativeScript projects via [CocoaPods](https://cocoapods.org/), a dependency manager for Swift and Objective-C Cocoa projects.
+
+## Table of Contents
+ * [Install CocoaPods](#install-cocoapods)
+ * [Create CLI Project](#create-cli-project)
+ * [Create NativeScript plugin with Podfile](#create-nativescript-plugin-with-podfile)
+ * [Build the Project](#build-the-project)
+
+## Install CocoaPods
+ You need to install CocoaPods. If you haven't yet, you can do so by:
+```
+$ sudo gem install cocoapods
+```
+> **NOTE:** All operations and code in this article are verified against CocoaPods 0.38.2.
+
+You can check your current version using:
+```
+$ pod --version
+```
+
+To update CocoaPods just install the gem again with:
+```
+sudo gem install cocoapods
+```
+
+## Create CLI Project
+To start, create a project, add the iOS platform.
+```bash
+$ tns create MYCocoaPods
+$ cd MYCocoaPods
+$ tns platform add ios
+```
+
+## Create NativeScript plugin with Podfile
+
+For more information see [NativeScript plugins](#PLUGINS.md)
+
+```
+cd ..
+mkdir my-plugin
+cd my-plugin
+touch package.json
+```
+
+Create a package.json file with the following content:
```

Comment:
Didn't the `touch package.json` command create the package.json file? If so, make this "In the newly created package.json file, insert the following syntax."